.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "css.lint.validProperties": ["composes"],
+  "editor.codeActionsOnSave": { "source.fixAll.eslint": true },
+  "editor.formatOnSave": true,
```

Comment:
I wonder if we should generally be setting "formatOnSave" or whether it should be a user setting.

Reply:
@chrislloyd was wondering the same thing - for now I kept it the same as what we use in Pinterest's codebase